client/src/components/CommentApp/comments.js

```diff
@@ -28,13 +29,13 @@ function getContentPath(fieldNode) {
     return '';
   }
   let element = fieldNode.closest('[data-contentpath]');
-  const contentPaths = [];
+  const contentpaths = [];
```

Comment:
**Question:** Are we suddenly re-branding "contentpath" to be a one word? :smile: Let's stay consistent everywhere if possible.

Reply:
Yeah, this was an effort to standardise all usages between this and the FE PR. Went with 'contentpath' as the model's original version, and changed everything else to match that.